wfs/1.0.0/spatial-wfs-v1_0_0-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/common/Wfs10JTStoGML200Converter.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+package org.codice.ddf.spatial.ogc.wfs.v1_0_0.catalog.common;
+
+import java.io.StringReader;
+
+import ogc.schema.opengis.gml.v_2_1_2.AbstractGeometryType;
+import ogc.schema.opengis.gml.v_2_1_2.GeometryCollectionType;
+import ogc.schema.opengis.gml.v_2_1_2.LineStringType;
+import ogc.schema.opengis.gml.v_2_1_2.MultiLineStringType;
+import ogc.schema.opengis.gml.v_2_1_2.MultiPointType;
+import ogc.schema.opengis.gml.v_2_1_2.MultiPolygonType;
+import ogc.schema.opengis.gml.v_2_1_2.ObjectFactory;
+import ogc.schema.opengis.gml.v_2_1_2.PointType;
+import ogc.schema.opengis.gml.v_2_1_2.PolygonType;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.xml.bind.JAXB;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.namespace.QName;
+
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.gml2.GMLWriter;
+
+public class Wfs10JTStoGML200Converter {
+
+    protected static final Logger LOGGER = LoggerFactory.getLogger(Wfs10JTStoGML200Converter.class);
+
+    private static final ObjectFactory gmlObjectFactory = new ObjectFactory();
+
+    private static final String GEOMETRYCOLLECTION_GML_INVALID_START = "<gml:MultiGeometry xmlns:gml='http://www.opengis.net/gml'>";
+
+    private static final String GEOMETRYCOLLECTION_GML_INVALID_END = "</gml:MultiGeometry>";
+
+    private static final String GEOMETRYCOLLECTION_GML_START_TAG = "<gml:GeometryCollection xmlns:gml='http://www.opengis.net/gml'>";
+
+    private static final String GEOMETRYCOLLECTION_GML_END_TAG = "</gml:GeometryCollection>";
+
+    public static String convertGeometryToGML(Geometry geometry) throws JAXBException {
+        GMLWriter gmlWriter = new GMLWriter(true);
+        String gml = gmlWriter.write(geometry);
+        return gml.replaceAll("\n", "");
```

Comment:
I stripped them out because they were causing XMLUnit to fail.  I can leave them in and just make the tests check for 'similar'.